README.md

```diff
@@ -20,20 +20,19 @@
 - Start the Operator locally
   ```bash
   $ oc new-project my-spring-app
-  $ OPERATOR_NAME=component-operator WATCH_NAMESPACE=my-spring-app KUBERNETES_CONFIG=$HOME/.kube/config go run cmd/component-operator/main.go
+  $ OPERATOR_NAME=component-operator WATCH_NAMESPACE=my-spring-app KUBERNETES_CONFIG=$HOME/.kube/config go run cmd/manager/main.go
   
 - In a separate terminal a component's yaml file with a project `/path/to/project`
-  ```bash
-  $ echo component.yml << EOF
-  apiVersion: component.k8s.io/v1alpha1
-  kind: Component
-  metadata:
-    name: my-spring-boot
-  spec:
-    deploymentMode: innerloop
-    EOF
-  $ oc apply -f component.yml 
-  ```  
+```bash
+echo "
+apiVersion: component.k8s.io/v1alpha1
+kind: Component
+metadata:
+  name: my-spring-boot
+spec:
+  runtime: spring-boot
+  deploymentMode: innerloop" | oc create -f -
```

Comment:
perhaps `oc create -f -` should be `oc apply -f -`?

Reply:
Yes @geoand we could use `apply` instead of `create` to allow update.
Testing update with:
```
echo "
apiVersion: component.k8s.io/v1alpha1
kind: Component
metadata:
  name: my-spring-boot
spec:
  runtime: spring-boot
  deploymentMode:outerloop" | oc apply -f -
```
moving from innerloop to outerloop build I run into (the only change watched by the operator):
```
INFO[0049] ### DeploymentConfig build creation failed
ERRO[0049] Outerloop creation failedDeploymentConfig.apps.openshift.io "my-spring-boot-build" is invalid: spec.triggers[0].imageChangeParams.from.name: Invalid value: ":latest": must match "[a-z0-9]+(?:[._-][a-z0-9]+)*"
```
Is it the intent or de outerloop template should not use `latest`?

Reply:
Not sure to tell you the truth, I haven't used it in a while. @cmoulliard should have all the answers :)

Reply:
It looks like that there is an error within the code and template (`
spec.triggers[0].imageChangeParams.from.nam
invalid: spec.triggers[0].imageChangeParams.from.name: Invalid value: ":latest": must match "[a-z0-9]+(?:[._-][a-z0-9]+)*`). I will have a look

Reply:
`apply` and `create` are equivalent. `apply` is perhaps a better verb to be used ;-)

Reply:
> Is it the intent or de outerloop template should not use `latest`?

Can you open an issue please and explain step by step what you did ? @corinnekrych

Reply:
@cmoulliard here is: https://github.com/snowdrop/component-operator/issues/36